thavalon-server/src/lobby/lobby_impl.rs

```diff
@@ -0,0 +1,212 @@
+use super::client::{OutgoingMessage, PlayerClient};
+use super::{LobbyChannel, LobbyCommand, LobbyError, LobbyResponse, ResponseChannel};
+use crate::database::games::{DBGameError, DBGameStatus, DatabaseGame};
+use crate::game::builder::GameBuilder;
+use crate::utils;
+
+use tokio::{
+    sync::mpsc::{self, Receiver},
+    task,
+};
+use warp::filters::ws::WebSocket;
+
+use std::collections::HashMap;
+
+/// A lobby for an individual game. The Lobby acts as an interface between the
+/// Thavalon game instance, the DatabaseGame which keeps the game state in sync
+/// with the database, and all players connected to the game.
+pub struct Lobby {
+    database_game: DatabaseGame,
+    friend_code: String,
+    players: HashMap<String, String>,
```

Comment:
can it be clarified what players and clients map to and from?

Reply:
Updated the players field to be more clear. I kept clients as is for now. I can clarify that as well if you think it's still confusing after the "players" field change.